app/controllers/application_controller.rb

```diff
@@ -1,4 +1,8 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  def current_user
+    # will be available in all controllers
+    @current_user = User.find(1)
```

Comment:
Kindly update this line t `User.first`. Models can be filtered by adding such ruby methods such as first, second, last, etc.

Reply:
@brytebee ok

Reply:
- [ ] [REQUIRED]n Hello @touredri , as suggested by the previous reviewer, let's update the code for the `current_user` method to: 

```ruby
def current_user
  User.first
end
```

The difference is that your code ` @current_user = User.find(1)` specifically finds a User record with ID 1 and stores it in the @current_user variable. In contrast, the code we recommend only retrieves the first User record from the database but does not store it for future reference.

Reply:
oh I done but not added to commit